test/libsolidity/semanticTests/emit_multiple_outputs.sol

```diff
@@ -0,0 +1,12 @@
+contract C {
+  event Terminated();
+
+  function terminate() external {
```

Comment:
@cameel did we not have a single test that would emit two of the same events before (in a sequence)? I would have expected @Andy53's change to have altered some other tests.